lib/Doctrine/Persistence/Mapping/Driver/AnnotationDriver.php

```diff
@@ -239,9 +240,11 @@ public function getAllClassNames()
         $declared = get_declared_classes();
 
         foreach ($declared as $className) {
-            $rc         = new ReflectionClass($className);
+            $rc = new ReflectionClass($className);
+
```

Comment:
With how often you inserted blank lines to break alignment I'd like to bring https://github.com/doctrine/coding-standard/pull/107 to your attention 😉

Reply:
I actually like that rule. The problem imo is caused by grouping lines of code that have no logical reason to be grouped. If you have lines that are grouped together, aligning them makes sense. Example of lines of code that make sense to be grouped and aligned:

```
$something1 = $input->getOption('something1');
$option2    = $input->getOption('option2');
$option3    = $input->getOption('option3');
```

Since they are all the same type of operation, I think it makes sense to group them and align them. I tend to put new lines inbetween statements that don't make sense to logically group so that the code has some breathing room and it is easier to read. Of course, this is subjective and is only how I feel. I don't know if others think this way as well.